src/containers/Header/HelpButton.js

```diff
@@ -1,22 +1,59 @@
-import React from 'react';
-import { connect } from 'react-redux';
+import React, { useState } from 'react';
+import { useTranslation } from "react-i18next";
+import ReactMarkdown from 'react-markdown';
+import rehypeRaw from 'rehype-raw';
+import { useSelector } from 'react-redux';
 
-import { NavLink } from 'reactstrap';
+import './header.scss';
```

Comment:
@Aringoaway Why it has to be imported here? I thought that it can be imported just to `index.js` of Header, since it should be propagated also to children of the header, thus also to the help button